### ISSUE TITLE
NN architecture

### ISSUE THREAD

**Issue by charlesoblack** [AlphaZeroIncubator/AlphaZero#4]
Our central NN architecture should be based off of AlphaZero but maybe we can leave some things flexible.

For instance, we could have a param for number of residual blocks; a parameter for what game to train/test on; a param for batch norm on/off (or maybe other norm types); and a param for which nonlinearity to use (ReLU to start with, like in the paper maybe?).

**Comment by charlesoblack** [comment]
@abhon I'd still like to have the network be one contiguous thing, please :)

**Comment by abhon** [comment]
Yeah I'll work on it now.

**Comment by charlesoblack** [comment]
Closed via #32
